okhttp/src/main/java/io/grpc/transport/okhttp/OutboundFlowController.java

```diff
@@ -94,13 +94,15 @@ synchronized void windowUpdate(int streamId, int delta) {
       writeStreams();
     } else {
       // Update the stream window and write any pending frames for the stream.
-      OutboundFlowState state = stateOrFail(streamId);
-      state.incrementStreamWindow(delta);
+      OutboundFlowState state = stateOrFailStranger(streamId);
```

Comment:
I don't like the idea of us throwing an exception like this. The approach in #443 is much better.